src/test/java/uk/ac/ebi/eva/pipeline/io/writers/VariantMongoWriterTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.io.writers;
+
+import com.googlecode.junittoolbox.ParallelRunner;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCollection;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.opencb.biodata.models.variant.Variant;
+import org.opencb.datastore.core.ObjectMap;
+import org.springframework.data.mongodb.core.MongoOperations;
+import uk.ac.ebi.eva.pipeline.model.converters.data.VariantToMongoDbObjectConverter;
+import uk.ac.ebi.eva.test.utils.JobTestUtils;
+import uk.ac.ebi.eva.utils.MongoDBHelper;
+import java.net.UnknownHostException;
+import java.util.*;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+
+/**
+ * Testing {@link VariantMongoWriter}
+ */
+@RunWith(ParallelRunner.class)
```

Comment:
This will not be necessary in the long term, as it can be set in the Maven POM file.